fyndiq/fyndiq-ui
# New Component : Productlist

# Overview

This PR adds 2 new components in 1 package: a product list and a wrapper. Have a look at the [README.md](https://github.com/fyndiq/fyndiq-ui/blob/29ac82ae5954dba9f3e60f6d1a7542fd105e4b43/packages/fyndiq-component-productlist/README.md) file for the package to understand how to get it working

![productlist](https://user-images.githubusercontent.com/1416801/28314845-73e6fabc-6bbc-11e7-901d-6b8fdb26cb2c.gif)

### Minor changes

- **fyndiq-component-price**: Change the styling for the price to disable wrapping
- **fyndiq-component-stars**: Change the propType for `rating`: allow string type. This is due to a limitation in our internal API where rating is often a string.
- **fyndiq-styles-colors**: Add a bright yellow color that matches Fyndiq's Daily Deals

### Line changes breakdown

- **400** lines of snapshots
- **250** lines of components code
- **160** lines of tests
- **140** lines of styles

# How to test

As usual, run the following steps:
1. `npm run bootstrap && npm run dev` and go to `localhost:6006` to see the Storybook.
